iot/api-client/manager/pom.xml

```diff
@@ -53,6 +55,11 @@
       <version>1.22.0</version>
     </dependency>
     <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>23.0</version>
+    </dependency>
+    <dependency>
       <groupId>com.google.api-client</groupId>
```

Comment:
can you confirm this is needed ? given you do include a `google-cloud-java` library:
https://github.com/GoogleCloudPlatform/google-cloud-java/blob/master/pom.xml#L750

Reply:
The tests will not work without explicitly including this.

Reply:
Updated, tests work still :)